### ISSUE TITLE
Setup instructions out of date

### ISSUE THREAD

**Issue by caseyyee** [MozVR/mozvr-site#177]
Should probably just redirect users to webvr.rocks for setup instructions.

**Comment by cvan** [comment]
I believe you, but which parts are out of date? I just updated them last week. (And, yes, agreed re: WebVR.Rocks.)

**Comment by caseyyee** [comment]
Yeah, sorry.   Was messaged to me while I was in a rush.  Had to make sure to file so I didn't forget ;)

- VIVE no longer requires the openvr.dll 
- Should be more clear between desktop & mobile support with Chrome using Daydream and Samsung Gear VR.

**Comment by cvan** [comment]
Gotcha, will update/remove - thanks!

**Comment by jmunsch** [comment]
https://github.com/MozVR/mozvr-site/blob/5467351cf21253ed9b456dfa121ae53c2629e36d/index.html#L108-L113

https://mozvr.com/

under the HTC vive tab?
